content/en/blog/_posts/2025-04-17-volume-populators-ga/index.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: "Kubernetes 1.33: Volume Populators Graduate to GA"
+date: 2025-04-17
+slug: volume-populators-ga
+author: >
+  Danna Wang (Google)
+  Sunny Song (Google)
+---
+
+The volume populators feature is now entering GA! The `AnyVolumeDataSource` feature
+gate defaults to enabled in Kubernetes v1.33, which means that users can specify any custom resource
+as the data source of a PVC.
+
+## What is new
+
+There are three major enhancements from beta.
+
+### Populator Pod is optional
+
+During the beta phase, we discovered that initiating data population doesn't always require a dedicated populator pod. In some cases, it's more efficient to interact with the cloud provider's APIs directly from the controller pod.
```

Comment:
Other than Cloud Providers, there could be other storage or backup vendors or other applications that may want to integrate with Volume Populators.